recipe/run_conda_forge_build_setup_osx

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# 1 core available on Travis CI OS X workers: https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments
+# MAKEFLAGS is passed through conda build: https://github.com/conda/conda-build/pull/917
+export MAKEFLAGS="-j1 ${MAKEFLAGS}"
+
+export PYTHONUNBUFFERED=1
+
+conda config --set show_channel_urls true
+conda config --set add_pip_as_python_dependency false
+
+conda update -n root --yes --quiet conda conda-build
+conda install -n root --yes --quiet jinja2 anaconda-client
+
+# Workaround an encoding bug in conda-build 1.20.1 when fixing shebangs.
+# See this issue ( https://github.com/conda/conda-build/issues/889 ).
+conda install -n root --yes --quiet conda-build=1.20.0
+
+conda info
```

Comment:
We were not running this on Travis CI previously, but I see no problem adding it.